Jenkinsfile

```diff
@@ -49,14 +49,27 @@ node {
 onlyOnMaster {
     milestone()
     stage("qa deploy") {
-        deployApp(image: img, app: "lms", env: "qa")
+        deployApp(image: img, app: "lms", env: "qa", region: "us-west-1")
     }
 
     milestone()
-    stage("prod deploy") {
-        input(message: "Deploy to prod?")
-        milestone()
-        deployApp(image: img, app: "lms", env: "prod")
+    stage("approval") {
+        input(message: "Proceed to production deploy?")
+    }
+
+    milestone()
+    stage("prod Deploy") {
+        lock("prod deploy") {
+            parallel(
+                us: {
+                    sleep 2
+                    deployApp(image: img, app: "lms", env: "prod", region: "us-west-1")
+                },
+                ca: {
+                    deployApp(image: img, app: "lms-ca", env: "prod", region: "ca-central-1")
+                }
+            )
+        }
```

Comment:
A stage dedicated to production deploys:

1. A `parallel` block has been introduced. When called, it will make two calls to `deployApp`. The first deals with deploying `us-west-1` and the second deals with deploying `ca-central-1`.
2. The `sleep 2` command in the `us` block is required to insert a small pause in the calls to `deployApp`. Without a pause it can end up running the first job twice.